src/todos.js

```diff
@@ -0,0 +1,37 @@
+let express = require('express');
+let bodyParser = require('body-parser');
+
+let app = express();
+app.use(bodyParser.json());
+
+let todos = [{id: 'jkhsdjkf', content: 'review this code'}];
+
+app.post('/todos', (req, res) => {
+  todos.push({
+    ...req.body,
+    id: Math.random().toString(32).slice(2)
+  });
+  res.sendStatus(201);
+});
+
+app.put('/todos/:id', (req, res) => {
+  todos[Number(req.params.id)] = req.body;
+  res.sendStatus(200);
+});
+
+app.get('/todos/:id', (req, res) => {
+  res.send(todos[id]);
+});
+
+app.get('/todos/all', (req, res) => {
+  res.send(todos);
```

Comment:
Return JSON using the [`json()` response method](https://expressjs.com/en/api.html#res.json).